jq/step4_if_fn_do.jq

```diff
@@ -0,0 +1,220 @@
+include "reader";
+include "printer";
+include "utils";
+include "interp";
+include "env";
+include "core";
+
+def read_line:
+    . as $in
+    | label $top
+    | _readline;
+
+def READ:
+    read_str | read_form | .value;
+
+# def eval_ast(env):
```

Comment:
Why that big block of commented out code?

Reply:
Ah, that reminds me to clean up.